### DISCUSSION TITLE
Domains - Invalid Configuration

### DISCUSSION THREAD

**Discussion by etaytch** [vercel/community#2575] [Help]
### Summary

I'm trying to configure the my domain information using namecheap.com.

I'm keep getting this error message on the Domains tab on Vercel:

<img width="904" alt="image" src="https://github.com/vercel/community/assets/11454939/be4f0850-e8bf-45ea-89a6-2904029a34b7">

I've added an 'A' Record of "76.76.21.21" on my namecheap.com account, but I can't find any 'A' Record of "162.255.119.73".

Please advise,
Etay

**Comment by smaeda-ks** [comment]
@etaytch 

It looks like `162.255.119.73` indeed exists in your DNS:

```
# (masked your domain)
$ dig +noall +ans yxxxxr.com @8.8.8.8
yxxxxr.com.		1799	IN	A	76.76.21.21
yxxxxr.com.		1800	IN	A	162.255.119.73
```

Could you double check your configuration? If you can't find that record, please contact namecheap support for further help! That's something they should be able to assist
